packages/api/pages/api/health.ts

```diff
@@ -1,5 +1,10 @@
 import { NextApiRequest, NextApiResponse } from 'next'
+import { client } from '../../db'
 
-export default function(req: NextApiRequest, res: NextApiResponse) {
-  return res.status(200).send('ok')
+export default async function(req: NextApiRequest, res: NextApiResponse) {
+  const collections = await client.db().listCollections().toArray()
+  return res.status(200).json({
+    status: 'ok',
+    collections
```

Comment:
I assume we're just doing this temporarily for debugging in the MVP stage? It doesn't seem ideal to broadcast data straight from the DB on a non-authenticated API route.

Reply:
Yep, I just wanted something to demonstrate that the database connection was working.